src/Http/Controllers/CP/Users/PasswordController.php

```diff
@@ -10,8 +10,6 @@ class PasswordController extends CpController
 {
     public function update(Request $request, $user)
```

Comment:
```suggestion
    public function update(Request $request, User $user)
```
If you remove the find you will have to type-hint the User i assume